### ISSUE TITLE
Inline CSS so that users don't have to include barebone css manually

### ISSUE THREAD

**Issue by lintiwen** [Fauntleroy/react-simple-dropdown#31]
I think it might be easier for users of this module, if we use the CSS-in-JS approach to inline the required css?
Any thoughts on doing or not-doing this?

**Comment by Phenominal** [comment]
I think that is a great idea. Currently I am running into problems with importing the CSS because of Webpack...

**Comment by sidonaldson** [comment]
I agree with @lintiwen but only because the barebone styles are so minimal!
